index.html

```diff
@@ -32,13 +32,15 @@
 				<a href="#">My Logo</a>
 			</div>
 			<section class="menu">
-				<nav><img src="Assets/images/Union.png" alt="Menu" class="hamburger">
+				<nav id="Main-menu"><img src="Assets/images/Union.png" alt="Menu" class="hamburger" id="hbg">
 					<!-- responsive web design for desktop screen head section-->
-					<ul class="nav-menu">
-						<li><a href="#">Portfolio</a></li>
-						<li><a href="#">About</a></li>
-						<li><a href="#">Contact</a></li>
+					<ul class="nav-menu" id="drop-down">
```

Comment:
- [ ] On desktop size the hamburger menu button should not be visible.

![Screenshot_20210623-022317_Chrome](https://user-images.githubusercontent.com/58504655/123001450-30c4f200-d3ca-11eb-9ade-25677e53c033.jpg)


- [ ] When the user clicks (or taps) the hamburger button, the mobile menu should appear.
- [ ] When the user clicks (or taps) the close (X) button, the mobile menu should disappear.
- [ ] When the user clicks (or taps) any option of the mobile menu, the mobile menu should disappear.